parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -378,6 +383,11 @@ try
             -KubeletNodeLabels $global:KubeletNodeLabels `
             -UseContainerD $useContainerD
 
+        if ($global:EnableCsiProxy) {
+            New-CsiProxyService -CsiProxyPackageUrl $global:CsiProxyUrl -KubeDir $global:KubeDir

```

Comment:
Could we please add a note in the README on where the csi proxy logs can be found in default setup.